plugins/window/src/scripts/hotkey.js

```diff
@@ -0,0 +1,6 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
```

Comment:
do we really need this hotkey script? seems like it could be replaced with a simple `keydown` event handler

Reply:
Fixed it